seahorse-gpgme-keyring.c: In function ‘seahorse_gpgme_keyring_finalize’:
seahorse-gpgme-keyring.c:862:1: error: size of array ‘_GStaticAssertCompileTimeAssertion_12’ is negative

--- origsrc/seahorse-3.4.1/pgp/seahorse-gpgme-keyring.c	2012-01-27 07:50:26.000000000 -0600
+++ src/seahorse-3.4.1/pgp/seahorse-gpgme-keyring.c	2012-05-06 04:54:05.214835400 -0500
@@ -859,7 +859,7 @@ seahorse_gpgme_keyring_finalize (GObject
 {
 	SeahorseGpgmeKeyring *self = SEAHORSE_GPGME_KEYRING (object);
 
-	g_clear_object (self->pv->actions);
+	g_clear_object (&self->pv->actions);
 	g_hash_table_destroy (self->pv->keys);
 
 	/* All monitoring and scheduling should be done */
